shm_wrapper/shm_wrapper.h

```diff
@@ -322,6 +338,6 @@ int log_data_write(char* key, param_type_t type, param_val_t value);
  *    types: array that will be filled with the parameter types, up to `num_params`
  *    values: array that will be filled with the parameter values, up to `num_params`
  */
-void log_data_read(uint8_t* num_params, char names[UCHAR_MAX][64], param_type_t types[UCHAR_MAX], param_val_t values[UCHAR_MAX]);
+void log_data_read(uint8_t* num_params, char names[UCHAR_MAX][LOG_KEY_LENGTH], param_type_t types[UCHAR_MAX], param_val_t values[UCHAR_MAX]);
```

Comment:
Last change! The comment for the argument `names` should be updated, similar to the one in `log_data_write()`. Something along the lines of `at most LOG_KEY_LENGTH characters long, including null-terminator`?

Reply:
It doesn't need it. There's no pointers here, just statically allocated arrays. this function will never fail unlike the write function

Reply:
Even though it will never fail, the comment is outdated, right? It still says `63` and it's better to refer to the `#define` that it's actually referring to, in the case that it will be changed. (I meant to comment this for line 337, but GitHub doesn't seem to allow me to add comments to unchanged lines.)

Reply:
Ok yea I was confused. I'll fix it